content/en/apps/guides/messaging/sms-recipients.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "SMS message recipients"
+linkTitle: "SMS Recipients"
+weight: 1
+description: >
+  Overview of the possible SMS recipient resolutions
+relatedContent: >
+  apps/features/messaging
+  apps/reference/app-settings/sms
+  apps/guides/data/hydration
+keywords: messaging
+---
+
+An outgoing SMS message configuration has the following fields:
+
+|property|description|required|
+|-------|---------|----------|
+|`translation_key`|The translation key of the message to send out. Available in 2.15+.|yes|
+|`messages`| Array of message objects, each with `content` and `locale` properties. From 2.15 on use `translation_key` instead.|no|
+|`recipient`| Recipient of the message.|no|
+
+### `recipient` values and resolutions:
+
+|value|resolves to|
+|-----|-----------|
+|*empty*| submitter |
+|reporting_unit| submitter | 
+|parent| primary contact of the patient's/submitter's place's parent (`patient.parent.parent.contact`) | 
+|grandparent| primary contact of the patient's/submitter's place's grandparent (`patient.parent.parent.parent.contact`) |
+|clinic| primary contact of the `clinic` in the patient's/submitter's lineage | 
+|health_center| primary contact of the `health_center` in the patient's/submitter's lineage | 
+|district| primary contact of the `district_hospital` in the patient's/submitter's lineage |
+|`ancestor:<contact_type>`| primary contact of the place of the requested type in the patient's/submitter's lineage |
+|`link:<tag>`| Linked doc that has requested `tag` in the patient's / submitter's lineage (direct mapping, not to primary contact). *As of 3.10.x*| 
+|`link:<contact_type>`| primary contact of the place of the requested `contact_type` in the patient's/submitter's lineage. *As of 3.10.x* | 
+| *custom object path* | a direct object path in the **context object** eg: `patient.parent.contact.other_phone` | 
+| *valid phone number* | requested phone number |
```

Comment:
Awesome explanation, thanks!

I think all of the hardcoded strings in the "value" column (eg: "clinic") should be wrapped in backticks so it's clear that it's a keyword.